plugins/rich-editor/src/scripts/flyouts/EmbedFlyout.tsx

```diff
@@ -143,4 +147,9 @@ export default function EmbedFlyout(props: IProps) {
             </DropDown>
         </>
     );
+
+    function handleVisibilityChange() {
```

Comment:
Can we move this up above the return statement? I know it get's hoisted, but it just feels really weird to have function definitions after a return statement.